src/Components/OrganisationsList/OrganisationsList.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './OrganisationsList.scss';
+import OrganisationCard from '../OrgCard/OrganisationCard';
+
+/**
+ * Component for displaying a list of organisations based on filters.
+ * @param {Object} props - The component props.
+ * @param {Array} props.organisations - The list of organisations to display.
+ * @param {string} props.default_image - The default image URL.
+ * @param {Object} props.filters - The filters to apply on the organisations.
+ * @returns {JSX.Element} - The rendered component.
+ */
+function OrganisationsList({ organisations, default_image = 'https://firebasestorage.googleapis.com/v0/b/cosh-website.appspot.com/o/COSH%20website%20assets%2FEvents%20images%2Fdefault-event-image.svg?alt=media&token=9756c350-dea2-42f4-a94d-00749c733968', filters }) {
```

Comment:
Again, this link hardcoding is not a very good approach.
Either store this image in the public directory or (if its not possible to do that), define this constant somewhere else.